src/components/WwwFrame/TheFooter.vue

```diff
@@ -62,9 +62,8 @@
 				<div class="wide">
 					<p>
 						Kiva is a 501(c)3 U.S. nonprofit fueled by passionate people.
-						Founded in 2005, and based in San Francisco,
-						with offices in Nairobi and staff around the globe.
-						<router-link to="/donate/supportus">Donate to our operating expenses.</router-link>
+						Founded in 2005, and based in San Francisco, with offices in Bangkok, Nairobi, Portland and staff around the globe.
```

Comment:
@foochris Please drop the content after "...Nairobi," down to the next line. We have a line limit in our linter that is failing this build. (max 120 characters per line, including tabs)